docs/pages/upgrading/cloud-kubernetes.mdx

```diff
@@ -4,127 +4,111 @@ description: Provides instructions for upgrading Teleport Cloud agents that run
 ---
 
 This guide explains how to upgrade Teleport Cloud agents running on Kubernetes.
-On Teleport Cloud, Auth Service and Proxy Service upgrades are managed for you.
-To keep agents up to date, you can either enroll them in automatic updates or
-upgrade them manually using the method you used to install Teleport.
+To keep agents up to date, you can either perform:
+- [Automatic updates](#automatic-updates) (recommended)
+- [Manual updates](#manual-updates)
 
 ## Prerequisites
 
 - Familiarity with the [Upgrading Compatibility Overview](./overview.mdx) guide,
   which describes the sequence in which to upgrade components of your cluster.
 
-- Teleport Enterprise Cloud. You can determine the current version of these
-  services by running the following command, where `mytenant` is the name of
-  your Teleport Enterprise Cloud tenant. This requires the
-  [`jq` CLI tool](https://jqlang.github.io/jq/):
-
-  ```code
-  $ curl -s https://mytenant.teleport.sh/webapi/ping | jq '.server_version'
-  ```
-
 - (!docs/pages/includes/cloud/tctl-tsh-prerequisite.mdx!)
 
 - (!docs/pages/includes/tctl.mdx!)
 
-Teleport supports automatic agent updates for the `teleport-kube-agent` Helm
-chart. The [Automatic Update Architecture
-](../architecture/agent-update-management.mdx) guide describes how agent
-updating works. Automatic agent upgrades require:
+## Check for agent updates
```

Comment:
We use the "Step n/d." format in H2s in how-to guides so readers know how far along they are. I would remove the "Automatic updates" H2 and turn the steps within that into "Step" headings.